samples/Groovy/filenames/Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+jettyUrl = 'http://localhost:8081/'
+
+def servers
+
+stage 'Dev'
+node {
+    checkout scm
+    servers = load 'servers.groovy'
```

Comment:
Auxiliary library files are conventionally just named `*.groovy` and so require no special treatment.